src/ui/Heading/styles.module.css

```diff
@@ -0,0 +1,29 @@
+.font-size-primary {
+  font-size: var(--font-size-xl);
+  font-family: var(--font-family);
+}
+
+.font-size-secondary {
+  font-size: var(--font-size-l);
+  font-family: var(--font-family);
+}
+
+.font-weight-thin {
+  font-weight: var(--font-weight-thin);
+}
+
+.font-weight-light {
+  font-weight: var(--font-weight-light);
+}
+
+.font-weight-regular {
+  font-weight: var(--font-weight-regular);
+}
+
+.font-weight-medium {
+  font-weight: var(--font-weight-medium);
+}
+
+.font-weight-bold {
+  font-weight: var(--font-weight-bold);
+}
```

Comment:
So these styles wouild be perfect for a design system, but we now only what to document "kudo" specific titles: 

```

.primary { 
    font-size: var(--font-size-xl; 
    font-familiy: ... 
    color: .. 
    // etc
}

```